src/FirstSteps/RPG/Equipment.cs

```diff
@@ -0,0 +1,78 @@
+﻿using FirstSteps.RPG.Items;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using FirstSteps.RPG.Heroes;
+
+namespace FirstSteps.RPG
+{
+    public class Equipment
+    {
+        private List<Item> backPack = new List<Item>();
+        const int maxBackpack = 5;
+        const int _maxWeight = 40;
+        public int _coinsBag;
+        private Hero _hero;
+
+        
+        public bool TryAddItemToBackpack(Item item)
+        {
+            if (!CanHandleSpecialItem(item))
+            {
+                return false;
+            }
+            else if (backPack.Count <= maxBackpack && CountWeight() <= _maxWeight && item.Weight + CountWeight() <= _maxWeight)
```

Comment:
Only this part is related to equipment and should stay here. The rest can be in `AddItemToBackpack` because there you have access to `CanHandleSpecialItem`.
```
else if (backPack.Count <= maxBackpack && CountWeight() <= _maxWeight && item.Weight + CountWeight() <= _maxWeight)
 {
  backPack.Add(item);
  Console.WriteLine(item);
  return true;
}
 ```